libcaf_core/caf/defaults.hpp

```diff
@@ -32,6 +32,11 @@ constexpr parameter<T> make_parameter(std::string_view name, T fallback) {
   return {name, fallback};
 }
 
+/// Configures how many actions scheduled_actor::delay may add to the internal
+/// queue for scheduled_actor::run_actions before being forced to push them to
+/// the mailbox instead.
```

Comment:
I'm assuming that "to the mailbox" means that other messages will be processed before the delayed actions. The doc of `void delay(action what)` In `coordinator.hpp` says:

> Delays execution of an action until all pending actions were executed.

Does that still fit? Are other tasks in the mailbox included in  "all pending actions"?

Reply:
> Does that still fit?

Yep. Everything that's in `actions_` gets processed before taking the next message from the mailbox. So by pushing into the mailbox, we automatically handle those actions after whatever is in `actions_` at the moment.